syz-verifier/verf/verifier.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 syzkaller projelct authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+package verf
```

Comment:
What is this package intended to handle in the end?
It mixes things like manager config, ExecutorBin and program verification logic. I think these should be separate, program verification/comparison is pure data processing, this part should not be concerned with managers, configs, binary files locations on virtual machines, vm stop channels, etc. It should accept just a program and N results and produce a verification verdict. Consider, currently it's not possible to unit-test the main Verify entry point because that will require manager configs, channels and all the unrelated things.